project-base/app/tests/FrontendApiBundle/Functional/Cart/RoundingPriceInCartTest.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\FrontendApiBundle\Functional\Cart;
+
+use App\DataFixtures\Demo\PaymentDataFixture;
+use App\DataFixtures\Demo\ProductDataFixture;
+use App\DataFixtures\Demo\StoreDataFixture;
+use App\DataFixtures\Demo\TransportDataFixture;
+use App\Model\Product\Product;
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Model\Payment\Payment;
+use Shopsys\FrameworkBundle\Model\Pricing\Currency\Currency;
+use Shopsys\FrameworkBundle\Model\Pricing\Currency\CurrencyDataFactory;
+use Shopsys\FrameworkBundle\Model\Store\Store;
+use Shopsys\FrameworkBundle\Model\Transport\Transport;
+use Tests\FrontendApiBundle\Test\GraphQlTestCase;
+
+class RoundingPriceInCartTest extends GraphQlTestCase
+{
+    private Product $testingProduct;
+
+    /**
+     * @inject
+     */
+    private CurrencyDataFactory $currencyDataFactory;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->testingProduct = $this->getReference(ProductDataFixture::PRODUCT_PREFIX . 1, Product::class);
+    }
+
+    public function testRoundingPriceIsNullForFirstDomain(): void
+    {
+        $response = $this->getResponseContentForGql(__DIR__ . '/../_graphql/mutation/AddToCartMutation.graphql', [
+            'productUuid' => $this->testingProduct->getUuid(),
+            'quantity' => 1,
+        ]);
+        $data = $this->getResponseDataForGraphQlType($response, 'AddToCart');
+
+        $cartUuid = $data['cart']['uuid'];
+
+        $response = $this->getResponseContentForGql(__DIR__ . '/graphql/GetCart.graphql', [
+            'cartUuid' => $cartUuid,
+        ]);
+        $data = $this->getResponseDataForGraphQlType($response, 'cart');
+
+        $this->assertNull($data['roundingPrice'], 'Rounding price has to be null for first domain');
```

Comment:
I am afraid this might be confusing on a project that changes the demo data and uses CZK on the first domain

Reply:
Good point. I have skipped the test if the first domain currency is CZK – bd06555a66 – what do you think?